test/extended/cli/version.go

```diff
@@ -0,0 +1,106 @@
+package cli
+
+import (
+        "encoding/json"
+        "fmt"
+        "log"
+        "os"
+        "regexp"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/openshift/openshift-tests/test/extended/util"
+)
+
+var _ = g.Describe("[sig-cli] oc [Slow]", func() {
```

Comment:
HI, Per https://docs.google.com/document/d/1k761p65J0Ig81hwZWaw73QJV6SBa7kPxbjuTEBlBbF4/edit# name rules section, there is sub-team here. is oc the sub-team name? if it is, OK. if it is not, please add sub-team. thanks.

and should no test label [Slow] here, so please remove it. And, move it to g.It